frontend/next.config.js

```diff
@@ -1,9 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  experimental: {
-    appDir: true,
-  },
-  output: 'standalone',
-}
+	experimental: {
+		appDir: true,
```

Comment:
It looks like we're having a tab/spaces issue here (which I prefer tabs). Is it possible to add `prettier` or something of the sort to enforce a style guide?

Reply:
I also prefer tabs and already have the prettier extension on so I was more cautious to change existing formats. Before I made changes, the majority of the format was spaces so it is good to know we are using prettier as our convention. I will make changes to existing code.